src/main/resources/i18n/messages.properties

```diff
@@ -1135,8 +1168,11 @@ ProjectMetadataTemplates.fields={0} fields
 
 MetadataFields.title=Fields
 
-MetadataFieldsList.create=Create Template
-MetadataFieldsList.empty=No metadata has been added to this project.
+MetadataFieldsListManager.create=Create Template
+MetadataFieldsListMember.empty=No metadata has been added to this project.
+MetadataFieldsListManager.restrictions=Restrictions
+MetadataFieldsListManager.restrictions-help=Each Metadata Field can be set to only be viewed by certain project roles.  Manager's will be able to view all fields while the Collaborators will only be able to view the fields that are selected for that role.
```

Comment:
Should we add `i.e. A Collaborator with a metadata role of LEVEL 1 will only be able to view metadata fields set to LEVEL 1. Whereas a Collaborator with metadata role of LEVEL 2 will be be able to view metadata fields set to LEVEL 1 or LEVEL 2.`

Reply:
Did you want me to just update this in the main branch? Or make another branch to add it? @ericenns

Reply:
In this branch is good. I don't think we need sub branches now.

Reply:
Sounds good

Reply:
Sounds good

Reply:
Updated in fb60b1fb991e7bdcd3ca5cc4f37341075d15ff3f